plugins/generator-1.17.1/forge-1.17.1/mappings/entities.yaml

```diff
@@ -0,0 +1,478 @@
+_default:
+  - Zombie
+  - EntityType.ZOMBIE
+  - zombie
+_mcreator_prefix: "CUSTOM:"
+_mcreator_map_template: "@NAMEEntity.CustomEntity"
+EntityAgeable: AgeableMob
+EntityAmbientCreature: AmbientCreature
+EntityAnimal: Animal
+EntityAreaEffectCloud:
+  - AreaEffectCloud
+  - EntityType.AREA_EFFECT_CLOUD
+  - area_effect_cloud
+EntityArmorStand:
+  - ArmorStand
+  - EntityType.ARMOR_STAND
+  - armor_stand
+EntityArrow:
+  - Arrow
+  - EntityType.ARROW
+  - arrow
+EntityAxolotl:
+  - Axolotl
+  - EntityType.AXOLOTL
+  - axolotl
+EntityBat:
+  - Bat
+  - EntityType.BAT
+  - bat
+EntityBee:
+  - Bee
+  - EntityType.BEE
+  - bee
+EntityBlaze:
+  - Blaze
+  - EntityType.BLAZE
+  - blaze
+EntityBoat:
+  - Boat
+  - EntityType.BOAT
+  - boat
+EntityCat:
+  - Cat
+  - EntityType.CAT
+  - cat
+EntityCaveSpider:
+  - CaveSpider
+  - EntityType.CAVE_SPIDER
+  - cave_spider
+EntityChicken:
+  - Chicken
+  - EntityType.CHICKEN
+  - chicken
+EntityCod:
+  - Cod
+  - EntityType.COD
+  - cod
+EntityCow:
+  - Cow
+  - EntityType.COW
+  - cow
+EntityCreature: CreatureEntity

```

Comment:
I couldn't find this one in the class list, could you recheck if it still exists? Possibly replaced by something like `NeutralMob`